app/src/main/java/com/github/se/polyfit/ui/screen/settings/AccountSettingsScreen.kt

```diff
@@ -117,7 +118,13 @@ private fun AccountSettings(
           keyboardOptions = KeyboardOptions.Default.copy(keyboardType = KeyboardType.Number),
           singleLine = true,
           onValueChange = {
-            val update = it.toLongOrNull()
+            var update = it.toLongOrNull()
+
+            // If there is a Long overflow, the use the previous value
+            if (update == null && it.toBigDecimalOrNull() != null) {
```

Comment:
Is it useful to check if update is null?
```suggestion
            if (it.toBigDecimalOrNull() != null) {
```

Reply:
If you try to convert a huge string, for example, "10000000000000000000000000", to a number bigger than the largest possible Long, then the function `.toLongOrNull()` returns null. The issue that was happening is that the null value was interpreted as `infinity`, which looked out of place.